### PR TITLE
add code snippets for Stripe connect

### PR DESCRIPTION
**Pull request by jchen293**
add code snippets for Stripe connect in all languages and JSON response

### CODE REVIEW

**Comment by jchen293** [comment]
Putting this PR in draft until we release the Gg so I can bump the Go version to make build passes

**Review by nwithan8 — APPROVED** [review]

File: official/docs/ruby/current/referral-customers/add-payment.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_API_KEY']
+
+payment_method = EasyPost::Beta::Referral.add_payment_method(
+  'cus_123',
+  'ba_123',
```
Comment (Justintime50) [inline official/docs/ruby/current/referral-customers/add-payment.rb:7]: This needs `priority` param

File: official/docs/golang/current/referral-customers/add-payment.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	"os"
+
+	"github.com/EasyPost/easypost-go/v2"
+)
+
+func main() {
+	apiKey := os.Getenv("EASYPOST_API_KEY")
+	client := easypost.New(apiKey)
+
+	paymentMethod, _ := client.BetaAddPaymentMethod("cus_123", "ba_123")
```
Comment (Justintime50) [inline official/docs/golang/current/referral-customers/add-payment.go:14]: This needs `priority` param

File: official/docs/csharp/current/referral-customers/add-payment.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public class Examples
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            var client = new EasyPost.Client(apiKey);
+
+            PaymentMethod paymentMethod = await Client.Beta.Referral.AddPaymentMethod("cus_123", "ba_123");
```
Comment (Justintime50) [inline official/docs/csharp/current/referral-customers/add-payment.cs:17]: This needs `priority` param

**Review by Justintime50 — CHANGES_REQUESTED** [review]
All of these files titled `add-payment` need to be `add-payment-method`. Those are totally different.

File: official/docs/golang/current/referral-customers/add-payment.go (hunk 2 above)
Comment (jchen293) [reply official/docs/golang/current/referral-customers/add-payment.go:14]: This function doesn't need the `priority` param:https://github.com/EasyPost/easypost-go/blob/master/referral.go#L190

File: official/docs/ruby/current/referral-customers/add-payment.rb (hunk 1 above)
Comment (jchen293) [reply official/docs/ruby/current/referral-customers/add-payment.rb:7]: Not necessary, the default param is set to `primary` in ruby: https://github.com/EasyPost/easypost-ruby/blob/master/lib/easypost/beta/referral.rb#L118

Comment (Justintime50) [reply official/docs/ruby/current/referral-customers/add-payment.rb:7]: This is documentation. We need to show people all their options.

File: official/docs/golang/current/referral-customers/add-payment.go (hunk 2 above)
Comment (Justintime50) [reply official/docs/golang/current/referral-customers/add-payment.go:14]: Please patch in the v2.11.1 version of Go and then include the priority here as well.

Comment (jchen293) [reply official/docs/golang/current/referral-customers/add-payment.go]: resolved

File: official/docs/csharp/current/referral-customers/add-payment.cs (hunk 3 above)
Comment (jchen293) [reply official/docs/csharp/current/referral-customers/add-payment.cs]: added

File: official/docs/ruby/current/referral-customers/add-payment.rb (hunk 1 above)
Comment (jchen293) [reply official/docs/ruby/current/referral-customers/add-payment.rb]: added

**Review by Justintime50 — APPROVED** [review]
Thanks for doing this and sorry for the Go mess.

**Pull request merged**